sources/packages/forms/src/form-definitions/studentwelcomepage.json

```diff
@@ -156,21 +162,50 @@
                       "key": "html5",
                       "type": "htmlelement",
                       "input": false,
-                      "tableView": false
+                      "tableView": false,
+                      "hideLabel": true
                     },
                     {
                       "label": "Start application",
                       "action": "event",
                       "showValidations": false,
                       "leftIcon": "fa fa-edit",
+                      "disabled": true,
                       "tableView": false,
                       "key": "startApplication",
+                      "logic": [
+                        {
+                          "name": "Enable when SIN is Valid",
+                          "trigger": {
+                            "type": "simple",
+                            "simple": {
+                              "show": true,
+                              "when": "sinValidStatus.sinStatus",
+                              "eq": "1"
```

Comment:
Similar comment. We can just check for valid indicator to be true, keeping the definition of valid SIN outside the form.